code/modules/martial_arts/judo.dm

```diff
@@ -58,6 +58,11 @@
 	add_attack_logs(A, D, "Melee attacked with [src]")
 	return TRUE
 
+/datum/martial_art/judo/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(IS_HORIZONTAL(A))
+		return FALSE
```

Comment:
Also, can you use better variable names here? I have no idea if A or D are supposed to be the user or the victim.

Reply:
This is what all other martial arts use here haha, I'll change it over